drivers/sensor/lsm6dsl/lsm6dsl_shub.c

```diff
@@ -131,6 +213,18 @@ static struct lsm6dsl_shub_sens_list {
 	},
 #endif  /* CONFIG_LSM6DSL_EXT0_LIS2MDL */
 
+#ifdef CONFIG_LSM6DSL_EXT0_LIS3MDL
+	{
+		/* LIS2MDL */
+		.i2c_addr       = { 0x1E },
```

Comment:
Change comment to ```/* LIS3MDL */``` and, more important, change the i2c_addr to ```.i2c_addr = {0x1C, 0x1E}```, because the LIS3MDL i2c address can be configured thru SAD[1]. Does it make sens to you? I don't know why I missed it before...

Reply:
thanks. I've pushed the changes.